src/build/arg/mod.rs

```diff
@@ -55,7 +55,12 @@ type ValidatorOs = Rc<dyn Fn(&OsStr) -> Result<(), String>>;
 pub struct Arg<'help> {
     pub(crate) id: Id,
     pub(crate) name: &'help str,
+    #[deprecated(
+        since = "3.0.0",
+        note = "Please use `about` field instead"
+    )]
     pub(crate) help: Option<&'help str>,
```

Comment:
Btw, since we will be using `self.about` from now on, we can remove this field, right?

Reply:
Yes